examples/libc/main.go

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
-// SPDX-FileCopyrightText: 2022 The Ebitengine Authors
+// SPDX-FileCopyrightText: 2023 The Ebitengine Authors
```

Comment:
We don't have to update the year for an existing file. (This is the same rule as Go and Chromium)